pkgs/development/python-modules/pypdt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "PyPDT";
```

```suggestion
  pname = "pypdt";
```

The name should follow [PEP 503](https://nixos.org/manual/nixpkgs/stable/#contributing).